google_maps/public/js/src/google_maps_edit.js

```diff
@@ -2,21 +2,21 @@
 var googleMapsXBlockApp = googleMapsXBlockApp || {};
 
 /**
- * Changes the src attribute of embedded google map iframe
+ * Changes the src attribute of Embedded Google Map iframe
  */
 googleMapsXBlockApp.changeEmbeddedMapLocation = function(parameters) {
   $('#google-map').attr('src', 'https://www.google.com/maps/embed/v1/place?' + $.param(parameters));
 };
 
 /**
- * Toggles error message depending on the validation result
+ * Toggles error message depending on the second parameter
  * @param {object} element for which the error message should be displayed
- * @param {bool} validation result indicator
+ * @param {bool} value that determines whether to show or hide the error message
  */
-googleMapsXBlockApp.toggleErrorMessage = function(element, passedValidation) {
-  $('.save-button').toggleClass('is-disabled', !passedValidation);
-  element.toggleClass('invalid-input', !passedValidation)
-    .siblings('.error-message').toggleClass('hidden', passedValidation);
+googleMapsXBlockApp.toggleErrorMessage = function(element, show) {
+  $('.save-button').toggleClass('is-disabled', !show);
+  element.toggleClass('invalid-input', !show)
+    .siblings('.error-message').toggleClass('hidden', show);
```

Comment:
You should flip this, now when you call ``toggleErrorMessage(el, true)`` it won't show the error message. It should be the other way around.